tests/CLI/helper_tests.py

```diff
@@ -252,16 +252,19 @@ def test_init(self):
 class ResolveIdTests(testing.TestCase):
 
     def test_resolve_id_one(self):
-        resolver = lambda r: [12345]
```

Comment:
Any specific reason for changing this from a lambda function?

Reply:
No _real_ reason. The analysis was failing because of it though. Checking it now, it would have been better to skip the assignment and put the lambda expression as the argument itself. That is normally how the lambda is intended to be used.

Reply:
I have replace the code with lambdas again, but removed the assignment process it did not like. They are true anonymous functions created within the function calls.

Reply:
But in general, I just didn't want the analysis failing for another section of code.